packages/thirdweb/src/auth/verify-hash.ts

```diff
@@ -1,16 +1,18 @@
 import { equalBytes } from "@noble/curves/abstract/utils";
-import { type Signature, encodeDeployData, serializeSignature } from "viem";
+import {
+  type Signature,
+  encodeDeployData,
+  serializeSignature,
+  universalSignatureValidatorByteCode,
+} from "viem";
 import type { Chain } from "../chains/types.js";
 import type { ThirdwebClient } from "../client/client.js";
 import { eth_call } from "../rpc/actions/eth_call.js";
 import { getRpcClient } from "../rpc/rpc.js";
 import { fromBytes } from "../utils/encoding/from-bytes.js";
 import { type Hex, isHex } from "../utils/encoding/hex.js";
 import { toBytes } from "../utils/encoding/to-bytes.js";
-import {
-  universalSignatureValidatorAbi,
-  universalSignatureValidatorByteCode,
-} from "./constants.js";
+import { universalSignatureValidatorAbi } from "./constants.js";
```

Comment:
maybe viem can export this, too? so we have it from the same source?

Reply:
Yeah meant to ask them, but can we merge this in the meantime?

Reply:
yep!